src/components/Courses/Courses.jsx

```diff
@@ -33,10 +23,8 @@ const Courses = () => {
 
 	useEffect(() => {
 		(async () => {
```

Comment:
You do not need here IIFE
```suggestion
	dispatch(fetchCourses());
	dispatch(fetchAuthors());
```